### PR TITLE
Internal: Add playwright sanity tests for Carousel [ED-6915]

### PR DESCRIPTION
**Pull request by ShayHel**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/image-carousel.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../pages/wp-admin-page.js' );
+
+test( 'Image Carousel', async ( { page }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo ),
+		editor = await wpAdmin.useElementorCleanPost();
+
+  //close Navigator
+  await page.click('#elementor-navigator__close');
```
Comment (matipojo) [inline tests/playwright/sanity/image-carousel.test.js:10]: JSLInt

**Review by matipojo — CHANGES_REQUESTED** [review]
The test is failing because you need to attach the Linux image from the artifact

File: tests/playwright/sanity/carousel.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	// Act.
+	await editor.addWidget( 'image-carousel' );
+	
+ // Click [aria-label="Add Images"]
+  await page.locator('[aria-label="Add Images"]').click();
+
+  // Click text=Media Library
+  await page.click('text=Media Library');
+  // Upload A.png, B.png, C.png, D.png, E.png
+  await page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/A.jpg' );
```
[the remark this answers is not in the document]
Comment (ShayHel) [reply tests/playwright/sanity/carousel.test.js]: Done

File: tests/playwright/sanity/carousel.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  await page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/E.jpg' );
+ // Click text=Create a new gallery
+  await page.locator('text=Create a new gallery').click();
+  // Click text=Insert gallery
+  await page.locator('text=Insert gallery').click();
+    // Click #elementor-controls >> :nth-match(div:has-text("Additional Options"), 3)
+    await page.click('#elementor-controls >> :nth-match(div:has-text("Additional Options"), 3)');
+    // Disable AutoPlay
+    await page.selectOption('select', 'no');
+  expect(await editor.getPreviewFrame().locator('div.elementor-image-carousel-wrapper.swiper-container.swiper-container-initialized').screenshot()).toMatchSnapshot('carousel.png');
```
[the remark this answers is not in the document]
Comment (ShayHel) [reply tests/playwright/sanity/carousel.test.js]: I did the reduction of the baseline, but reducing the widget width I am afraid will affect the quality of the test.

File: tests/playwright/sanity/image-carousel.test.js (hunk 1 above)
Comment (ShayHel) [reply tests/playwright/sanity/image-carousel.test.js:10]: Done

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix the checks errors

**Pull request merged**